vyper/semantics/types/function.py

```diff
@@ -469,7 +470,23 @@ def fetch_call_return(self, node: vy_ast.Call) -> Optional[BaseTypeDefinition]:
                 if not isinstance(kwarg.value, vy_ast.NameConstant):
                     raise InvalidType("skip_contract_check must be literal bool", kwarg.value)
             else:
-                validate_expected_type(kwarg.arg, kwarg.value)
+                kwarg_pattern = fr"{kwarg.arg}\s*=\s*{re.escape(kwarg.value.node_source_code)}"
+                modified_line = re.sub(
```

Comment:
i don't generally like regexes for this kind of thing, they are like hand grenades in that their behavior is hard to predict. but as this seems to work even in edge cases - i'll allow it.

Reply:
Got it. I think an alternative is to create a copy of the AST node, modify it (moving the kwarg node from keywords to args), and then generate the source code from the modified AST node. However, this would require some sort of pretty print function for each node type.